### DISCUSSION TITLE
关于SEVI大作业数据格式的问题

### DISCUSSION THREAD

**Discussion by liqingyang1107** [physics-data/faq#88] [Q&A] [answered]
数据格式要求中说，每个HDF5文件中包含不多于10个表格，请问此处表格是二维表格还是数据集之类的？（实在无法想象把50000个的2048*2048的计数表格写到一个二维表格里）

**Comment by littlewu2508** [comment, marked as answer]
HDF5 可以存储多维的数据。提示：可以使用 numpy 复合数据类型。

**Comment by littlewu2508** [comment]
其实简单分析一下，几乎可以肯定图像信息要 3 维的了。不过其它辅助信息该如何组织，也需要一些设计
